tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -4536,8 +4540,15 @@ SpirvEmitter::incDecRWACSBufferCounter(const CXXMemberCallExpr *expr,
     return nullptr;
   }
 
+  llvm::SmallVector<SpirvInstruction *, 2> indexes;
+  if(const auto *arraySubscriptExpr = dyn_cast<ArraySubscriptExpr>(object)) {
+    //TODO: Handle multi-dimensional arrays.
+    indexes.push_back(doExpr(arraySubscriptExpr->getIdx()));
+  }
+  indexes.push_back(zero);
```

Comment:
Can you add a comment to explain why you push zero? I'd assume it's because the counter is at index zero, but might be unclear.